webapp/channels/src/components/onboarding_tasks/onboarding_video_modal/onboarding_video_modal.scss

```diff
@@ -43,15 +44,14 @@
         height: 100vh;
         max-height: 506px;
         padding: 0;
-        border: none;
-        border-radius: 8px;
+        border: none !important;
```

Comment:
This isn't ideal but i am sure you must have had a valid reason to use it here

Reply:
Agreed. It was kind of a lazy way to override. I've updated this to remove the `!important` rule in the latest commit